pkgs/tools/misc/pbincli/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchPypi
+, python3
+, runCommand
+, pbincli
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "PBinCLI";
+  version = "0.3.5";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
  pname = "pbincli";
  version = "0.3.5";
  format = "setuptools";

  src = fetchPypi {
    pname = "PBinCLI";
    inherit version;
```
see python language docs